app/flows/initial_request_flow.rb

```diff
@@ -51,14 +51,23 @@ def redirect?
     (training_provider_search? && one_search_result?) || training_provider_selected?
   end
 
+  def allocation
+    if (training_provider_search? && one_search_result?) || training_provider_selected?
+      Allocation.where(recruitment_cycle_year: recruitment_cycle.year)
```

Comment:
Worth moving this query to a scope?

Reply:
If you can point me in the direction of example of how this is done in json api client I will make the change

Reply:
Yeah forget that - mistaken this for the API